libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -932,8 +932,7 @@ void GCS_MAVLINK::send_radio_in(uint8_t receiver_rssi)
     mavlink_status_t *status = mavlink_get_channel_status(chan);
 
     uint16_t values[18];
```

Comment:
where does the magic number 18 come from ? Shouldn't it be a define ?

Reply:
This is a MAVLink message that is being filled in, there are 18 fields in the message. There is no way to get the number from MAVLink automatically, and given that the constructor needs to be setup by a list of entries I don't think a define would help readability at all here.